rpc_jobs/defaults.yml

```diff
@@ -8,6 +8,7 @@
     DEPLOY_SWIFT: "yes"
     DEPLOY_CEPH: "no"
     DEPLOY_ELK: "yes"
+    DEPLOY_SUPPORT_ROLE: "yes"
```

Comment:
I'd rather not set it, or set it to the same value as the default in the rpc-o branch. Because this is not branch specific, and we plan to backport the disabling of the support role in all roles, we should probably set that to no.